src/renderer/redux/middlewares/persistState.js

```diff
@@ -1,24 +1,14 @@
-import { throttle } from 'lodash';
-
 import { send } from 'services/ipc';
-import { SAVE_STATE } from 'common/events';
 
 /**
  * Redux middleware creator to save the state to a file everytime there is an action.
  *
- * @example const middleware = makeSaveState();
+ * @example
+ * const middleware = makeSaveState();
+ * const store = createStore(reducers, applyMiddleware(middleware));
  */
-export default () => {
-  // Only save the state every second if there are numerous actions being dispatched.
-  const throttledSaveState = throttle(
```

Comment:
Why did you remove the `throttle` ? Aren't you afraid to overload the disk by calling a write each time the state change ?

Reply:
I'm throttling on the main process instead, otherwise I cannot know if there are pending saves before exiting.

Reply:
I'm more concerned about IPC speed but if we start dispatching a 100 actions / s, even the UI is going to be slow ^^